source/deploy/server/containers/install-docker.rst

```diff
@@ -1,14 +1,9 @@
-Deploy Mattermost via Docker
-==============================
-
-.. include:: ../_static/badges/allplans-selfhosted.rst
-  :start-after: :nosearch:
-
 .. important::
-   This deployment method is not recommended for production environments. If you're looking to deploy a production Mattermost instance, we recommend using managed container services like AWS ECS, Azure Container Apps, or Google Cloud Run. These services provide better scalability, reliability, and maintenance features suited for production workloads.
+   - This deployment method isn't recommended for production environments. 
```

Comment:
What are the downsides of a docker deployment vs a tarball one? As a matter of fact, there are a lot of Docker deployments out there.

Reply:
There's a thread here: https://hub.mattermost.com/private-core/pl/3zey5kaq7bgtibpqoza4ujw36h discussing this